File: Python/program65.py
def main():
    iNo = 7236
    iDigit = 0

    # First digit
    
    while iNo != 0:
        
        print("--------------------------------------------------")
        iDigit = iNo % 10
        print(iDigit)
        iNo = iNo // 10
        print(iNo)
        print("--------------------------------------------------")


    
if __name__ == "__main__":
    main()

File: Java/program65.java
class program65
{
    public static void main(String A[])
    {
    int iNo = 7236;
    int iDigit = 0;

    while(iNo != 0)
    {
        System.out.println("--------------------------------------------------\n");
        iDigit = iNo % 10;
        System.out.println("iDigit is : %d\n"+iDigit);
        iNo = iNo / 10;
        System.out.println("iNo is : %d\n"+iNo);
    }
    
    System.out.println("--------------------------------------------------\n");

    
    }
}

